servicex/decorators.py

```diff
@@ -0,0 +1,43 @@
+from functools import wraps
+
+from flask import redirect, request, session, url_for, current_app, make_response
+from flask_jwt_extended import get_jwt_identity, jwt_required
+
+from servicex.models import UserModel
+
+
+def authenticated(fn):
```

Comment:
This change would have been better as its own PR, or at least isolated in a commit (along with the changes to the decorated routes)